kale/task.py

```diff
@@ -88,15 +88,24 @@ def _get_task_id(cls, *args, **kwargs):
         return '%s_uuid_%s' % (cls.__name__, uuid.uuid1())
 
     @classmethod
-    def publish(cls, app_data, *args, **kwargs):
+    def publish(cls, *args, **kwargs):
         """Class method to publish a task given instance specific arguments."""
+        return cls.publish_with_delay(None, *args, **kwargs)
+
+    @classmethod
+    def publish_with_delay(cls, delay_sec, app_data, *args, **kwargs):
+        """Class method to publish a task with a delay.
+
+        Args:
+            delay_sec: Number of seconds to delay the task.
```

Comment:
This is a strange beast: A required arg that happens to have (unspecified) default behavior if you pass in None.  Can you at least update the docs to reflect this?

    delay_sec: Number of seconds to delay the task.  If None, publish with the default delay (probably 0).